3.process_cfret_features/extract_sc_features.ipynb

```diff
@@ -61,18 +61,26 @@
     "# Set paths for plate localhost220512140003_KK22-05-198\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    # Set paths for plate localhost230405150001
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/CFReT_data/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>